db/seeds.rb

```diff
@@ -3,7 +3,13 @@
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 return unless Rails.env.development?
 
-FactoryGirl.create_list(:user, 10, :with_posts)
+@users = []
+10.times do
+  @users << User.create(FactoryGirl.attributes_for(:user, :with_posts))
```

Comment:
use for this reason `create_list` method

Reply:
i now that method, but then I use User.create, seeds will not falls, when we will have duplicate valuaes